### PR TITLE
Smart Wallets v2

### PR DESCRIPTION
**Pull request by emmaodia**

### CODE REVIEW

File: api-references/fuse-smart-wallet-v-api.yaml
Comment (NikolayRn) [inline api-references/fuse-smart-wallet-v-api.yaml:1]: Please rename .yaml file to api-references/fuse-smart-wallet-v2-api.yaml, in current name version is missing

File: docs/smart-wallet-api-v/auth.api.mdx
Comment (NikolayRn) [inline docs/smart-wallet-api-v/auth.api.mdx:1]: version here

**Review by NikolayRn — CHANGES_REQUESTED** [review]
I see that in every file there is no version in its names, please add it

File: api-references/fuse-smart-wallet-v-api.yaml
Comment (emmaodia) [reply api-references/fuse-smart-wallet-v-api.yaml:1]: Thank you for the feedback @NikolayRn Done

File: docs/smart-wallet-api-v/auth.api.mdx
Comment (emmaodia) [reply docs/smart-wallet-api-v/auth.api.mdx:1]: Thank you for the feedback @NikolayRn Done

**Review by NikolayRn — APPROVED** [review]
LGTM

**Pull request merged**